example/unikernel.ml

```diff
@@ -135,7 +133,7 @@ module Main
        send ARP traffic to the normal ARP listener and responder,
        handle ipv4 traffic with the functions we've defined above for NATting,
        and ignore all ipv6 traffic (ipv6 has no need for NAT!). *)
-    let listen_public = Public_net.listen public_netif (
+    let listen_public = Public_net.listen ~header_size:0 public_netif (
```

Comment:
in common networks, you'll want to pass `header_size:14` here and a few lines below (if you're on 802.1q 18) - this is used by the network device to allocate a receive buffer - i.e. it should be MTU + ethernet header size.